### PR TITLE
Added demoService and controller

### PR DESCRIPTION
**Pull request by JunboHe-David**

### CODE REVIEW

File: src/main/java/com/YouRL/controller/DemoController.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import com.YouRL.service.DemoService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("api")
+public class DemoController {
```
Comment (LeonQC) [inline src/main/java/com/YouRL/controller/DemoController.java:12]: Should have more meaningful class name

Comment (JunboHe-David) [reply src/main/java/com/YouRL/controller/DemoController.java:12]: This is only for the demo. It will be changed in the future.

**Comment by LeonQC** [comment]
Please add PR description and test evidence

**Comment by JunboHe-David** [comment]
Create demo service and controller
demo
r

**Review by LeonQC — APPROVED** [review]

**Pull request merged**